app/views/users/new.html.erb

```diff
@@ -0,0 +1,32 @@
+
+<%= form_for @user, url: {action: "create"} do |f| %>
+  User Name: <%= f.text_field :user_name, placeholder: "Only first name" %><br />
+  Email: <%= f.text_field :email, placeholder: "example@hotmail.com" %><br />
+  Password: <%= f.text_field :password, placeholder: "xxxxx" %><br />
+  <%= f.submit "Create" %>
```

Comment:
Line 4 should be `f.email_field` and line 5 should `f.password_field`

Make the above corrections in the` edit.html.erb`.